### PR TITLE
Updating references from Device Explorer to Azure IoT Explorer.

### PR DESCRIPTION
**Pull request by rafilho**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-c/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.
- I submitted this PR against the correct branch: 
  - [ ] This pull-request is submitted against the `master` branch. 
  - [ ] I have merged the latest `master` branch prior to submission and re-merged as needed after I took any feedback.
  - [ ] I have squashed my changes into one with a clear description of the change.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: iothub_client/samples/ios/CocoaPods-Samples.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `azure-iot-sdk-c/iothub_client/samples` directory some Azure IoT samples tha
 are appropriate for working with the iOS sample application:
 
 * **iothub_ll_c2d_sample** (cloud to device) This sample demonstrates sending messages
-from the cloud to the device. Device Explorer can be used to send the messages.
+from the cloud to the device. Azure IoT Explorer can be used to send the messages.
```
Comment (gregman-msft) [inline iothub_client/samples/ios/CocoaPods-Samples.md]: nit, should this be a link to the github and/or referenced as recommended in the prereqs for the doc?

Comment (rafilho) [reply iothub_client/samples/ios/CocoaPods-Samples.md]: I am just following the instructions we received from the team. I forwarded you the e-mail as FYI,

**Pull request merged**